var/spack/repos/builtin/packages/netcdf-fortran/package.py

```diff
@@ -31,8 +31,10 @@ class NetcdfFortran(Package):
     url      = "http://www.unidata.ucar.edu/downloads/netcdf/ftp/netcdf-fortran-4.4.3.tar.gz"
 
     version('4.4.3', 'bfd4ae23a34635b273d3eb0d91cbde9e')
+    variant('mpi', default=False, description='Enables MPI')
 
-    depends_on('netcdf')
+    depends_on('netcdf+mpi', when='+mpi')
```

Comment:
i have not checked `netcdf-fortran`, but this will not make `netcdf-fortran` explicitly depend on `mpi`. I think you could achieve the same result for `ferret` by declaring it to
```
depends_on("netcdf~mpi")
depends_on("netcdf-fortran~mpi")
```
which would not require extra variant here. I would say it would be a better way.